core/domain/user_jobs_continuous.py

```diff
@@ -549,21 +550,37 @@ def map(item):
         else:
             calculate_exploration_impact_score = False
 
-        statistics = (
-            stats_jobs_continuous.StatisticsAggregator.get_statistics(
-                item.id, stats_jobs_continuous.VERSION_ALL))
-        answer_count = 0
-        # Find number of users per state (card), and subtract no answer
-        # This will not count people who have been back to a state twice
-        # but did not give an answer the second time, but is probably the
-        # closest we can get with current statistics to "number of users
-        # who gave an answer" since it is "number of users who always gave
-        # an answer".
-        for state_name in statistics['state_hit_counts']:
-            state_stats = statistics['state_hit_counts'][state_name]
-            first_entry_count = state_stats.get('first_entry_count', 0)
-            no_answer_count = state_stats.get('no_answer_count', 0)
-            answer_count += first_entry_count - no_answer_count
+        if feconf.ENABLE_NEW_STATS_FRAMEWORK:
+            exploration_stats = stats_services.get_exploration_stats(
+                item.id, item.version)
+            answer_count = 0
+            # For each state, find the number of first entries to the state.
+            # This is approximately considered to be equal to number of users
+            # who answered the state because very few users enter a state and
+            # leave without answering.
+            for state_name in exploration_stats.state_stats_mapping:
+                state_stats = exploration_stats.state_stats_mapping[state_name]
+                first_hit_count = state_stats.first_hit_count
+                answer_count += first_hit_count
```

Comment:
Consider making the computation of answer_count a method on the ExplorationStats domain object.

Reply:
Done.